packages/forklift-console-plugin/locales/en/plugin__forklift-console-plugin.json

```diff
@@ -412,7 +412,7 @@
   "vSphere REST API user name.": "vSphere REST API user name.",
   "Warning concerns": "Warning concerns",
   "Warning: The provided URL does not end with \"ovirt-engine/api\". Ensure it includes the correct path, like: https://rhv.com/ovirt-engine/api.": "Warning: The provided URL does not end with \"ovirt-engine/api\". Ensure it includes the correct path, like: https://rhv.com/ovirt-engine/api.",
-  "Warning: The provided URL does not end with \"sdk\". Ensure it includes the correct path, like: https://vcenter.com/sdk.": "Warning: The provided URL does not end with \"sdk\". Ensure it includes the correct path, like: https://vcenter.com/sdk.",
+  "Warning: The provided URL does not end with the SDK endpoint path: \"sdk\". Ensure the URL includes the correct path. For example: https://vCenter-host-example.com/sdk": "Warning: The provided URL does not end with the SDK endpoint path: \"sdk\". Ensure the URL includes the correct path. For example: https://vCenter-host-example.com/sdk",
```

Comment:
nit-picking - we're not consistent here and also in #744 about the leading `/` in the path - here we don't have it (we say `sdk`) and below we have it (`/sdk`)